docs/getting-started.md

```diff
@@ -33,11 +33,19 @@ To load RDF data from a `.jelly` file:
 
 RDFLib will reconstruct the graph from the serialized Jelly stream.
 
+### Streaming graph parser
+
+To process a Jelly stream frame-by-frame, loading each as a separate RDFLib graph:
+
+{{ code_example('rdflib/03_parse_grouped.py') }}
+
+Because `parse_jelly_grouped` returns a generator, each iteration receives **one** graph, keeping memory usage bounded to the current frame. Thus, large datasets and live streams can be processed efficiently.
+
 ### File extension support
 
 You can generally omit the `format="jelly"` parameter if the file ends in `.jelly` – RDFLib will auto-detect the format:
 
-{{ code_example('rdflib/03_parse_autodetect.py') }}
+{{ code_example('rdflib/04_parse_autodetect.py') }}
```

Comment:
You don't have to rename this – I would advise against it, because it creates a mess in the repo. The numbers only specify the order of execution in tests. Otherwise you don't need to worry about these.